src/lib/caches/Twitch.js

```
@@ -0,0 +1,48 @@
+const WF_ARSENAL_ID = 'ud1zj704c0eb1s553jbkayvqxjft97';
+const TWITCH_CHANNEL_ID = '89104719'; // tobitenno
+
+const dirName = dirname(fileURLToPath(import.meta.url));
+
+export default class TwitchCache {
+  static #cache = create({ cacheId: '.twitch', cacheDir: resolve(dirName, '../../../') });
+
+  static async #hydrate(logger = Logger('TWITCH')) {
+    if (
+      CLIENT_ID &&
+      Date.now() - (this.#cache.getKey('last_updt') || 0) >= TWO_DAYS &&
```

Biome

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)